script/openqa-worker-cacheservice-minion

```diff
@@ -0,0 +1 @@
+exec $(dirname $0)/openqa-workercache minion worker -m production "$@"
```

Comment:
Is there any chance the `-m production` can be done in a config file instead?

Reply:
I don't think so. At least this is what I gather from https://github.com/os-autoinst/openQA/issues/2864